traits/observers/_list_change_event.py

```diff
@@ -0,0 +1,73 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+""" Event object for representing mutations to a list.
+"""
+
+# ListChangeEvent is exposed in the public API
+
+
+class ListChangeEvent:
+    """ Event object to represent mutations to a list.
+
+    Attributes
+    ----------
+    trait_list : list
```

Comment:
Can we be more precise with the type here? Will `trait_list` always be an instance of `TraitList`?